src/config.rs

```diff
@@ -476,6 +476,7 @@ pub struct TunableValues {
     pub message_user_color: bool,
     pub default_room: Option<String>,
     pub open_command: Option<Vec<String>>,
+    pub notifications: bool,
```

Comment:
Can you make this a String instead that can be set to `"all"` or `"none"`? That way it can be configurable to other values in the future, like `"mentions"`.

Reply:
Or rather, a string in the `config.json`, but an enum that derives `Deserialize` here.